backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -696,4 +696,12 @@ public static ModelDBServiceResourceTypes getModelDBServiceResourceTypesFromRepo
     }
     return modelDBServiceResourceTypes;
   }
+
+  public static String getStringFromProtoObjectSilent(MessageOrBuilder object) {
+    try {
+      return JsonFormat.printer().preservingProtoFieldNames().print(object);
+    } catch (InvalidProtocolBufferException e) {
+      throw new ModelDBException(e);
+    }
+  }
```

Comment:
this is already available at the ModelDBUtils.java. we can simply use it for silent purpose as well.

Reply:
Fixed